solutions/kubernetes/cortx-deploy.sh

```diff
@@ -49,7 +49,7 @@ fi
 
 function check_params() {
     if [ -z "$CORTX_SCRIPTS_REPO" ]; then echo "CORTX_SCRIPTS_REPO not provided. Using default: Seagate/cortx-k8s ";CORTX_SCRIPTS_REPO="Seagate/cortx-k8s"; fi
-    if [ -z "$CORTX_SCRIPTS_BRANCH" ]; then echo "CORTX_SCRIPTS_BRANCH not provided. Using default: v0.6.0";CORTX_SCRIPTS_BRANCH="v0.6.0"; fi
+    if [ -z "$CORTX_SCRIPTS_BRANCH" ]; then echo "CORTX_SCRIPTS_BRANCH not provided. Using default: v0.6.0";CORTX_SCRIPTS_BRANCH="v0.7.0"; fi
```

Comment:
Please change v0.6.0 > v0.7.0 in echo statement.

Reply:
Done